distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -28,46 +28,33 @@ karaf.systemBundlesStartLevel=50
 # You can place any customized configuration here.
 #
 
-#Using equinox until DDF and dependent applications can start appropriately
-#under felix
-#karaf.framework=felix
+# Framework selection properties
+#
+# Using equinox until DDF and dependent applications can start appropriately under felix
 karaf.framework=equinox
 
-# Add console delay, will only open after everything is loaded
+#
+# Don't delay the console startup. Set to true if you want the console to start after all other bundles
```

Comment:
This is copied from https://github.com/apache/karaf/blob/karaf-4.1.1/demos/web/src/main/webapp/WEB-INF/karaf/etc/config.properties#L384, but I'll fix this anyway:+1: